cmd/config/cluster-density-ms/cluster-density-ms.yml

```diff
@@ -8,11 +8,27 @@ global:
         - conditionType: Ready
           metric: P99
           threshold: {{.POD_READY_THRESHOLD}}
-indexers:
-  - esServers: ["{{.ES_SERVER}}"]
-    insecureSkipVerify: true
-    defaultIndex: {{.ES_INDEX}}
-    type: {{.INDEXING_TYPE}}
+metricsEndpoints:
+{{ if .ES_SERVER }}
+  - metrics: [{{.METRICS}}]
+    alerts: [{{.ALERTS}}]
+    indexer:
+      esServers: ["{{.ES_SERVER}}"]
+      insecureSkipVerify: true
+      defaultIndex: {{.ES_INDEX}}
+      type: {{.INDEXING_TYPE}}
+{{ end }}
+{{ if .LOCAL_INDEXING }}
```

Comment:
```suggestion
{{ if eq .LOCAL_INDEXING "true" }}
```

Reply:
We should check for the "true" string in the conditional

Reply:
Sounds good to me. To only enable when applied.